### PR TITLE
Update Module4-exercises.tex

### PR DESCRIPTION
**Pull request by qrn99**
Sorry I forgot to pull :(

Added 3 proof questions.

### CODE REVIEW

File: problemsets-mat223/modules/module4-exercises.tex [hunk 1]
```diff
@@ -181,5 +181,12 @@
 				real numbers.
 			\end{enumerate}
 		\end{solution}
+        
+		\prob Let $\vec u, \vec v \in \R^n$. Prove $|\vec u \cdot \vec v| \leq \|\vec u \|\|\vec v\|$, the \emph{Cauchy-Schwarz inequality}. Hint: use the geometric formula of the dot product of $\vec u, \vec v$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:185]: Use `\Norm{...}` and `\Abs{...}` . 

I think this problem needs a bit more scaffolding. Can you make it into a multi-part question?

File: problemsets-mat223/modules/module4-exercises.tex [hunk 2]
```diff
@@ -181,5 +181,12 @@
...
+
+	        \prob Let $\vec u, \vec v \in \R^n$. Prove $\|\vec u + \vec v\| \leq \|\vec u \| + \|\vec v\|$, the \emph{triangle inequality} in $\R^n$.
+	       Hint: use the \emph{Cauchy-Schwarz inequality}.
+
+	        \prob Suppose $\mathcal {A} = \{\vec v_1,\dots , \vec v_k\} \subset \R^n$ is a set of nonzero, mutually orthogonal vectors, i.e., for all $i \neq j$, $\vec v_i \cdot \vec v_j = 0$. Prove $\mathcal{A}$ is a linearly independent set.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:190]: Let's start with the definition and make this two-part. (a) just mutually orthogonal, (b) mutually orthogonal and non-zero.

Comment (qrn99) [reply problemsets-mat223/modules/module4-exercises.tex:190]: for part a), do we want the students to realize that if one of the vectors is the zero vector, $\mathcal {A}$ is not linearly independent? So I should word the question as: "Suppose $\mathcal{A}$ is a set of mutually orthogonal vectors. Is $\mathcal{A}$ a linearly independent set? Why or Why not?"

Comment (siefkenj) [reply problemsets-mat223/modules/module4-exercises.tex:190]: You got it!

**Comment by siefkenj** [comment]
Run this code through the latex beautifier: https://siefkenj.github.io/latex-parser/   so it line wraps.

File: problemsets-mat223/modules/module4-exercises.tex [hunk 3]
```diff
@@ -181,23 +181,48 @@
...
+		\prob Let $\mathcal{A}= \{\vec v_{1},\dots , \vec v_{k}\} \subset \R^{n}$.
+		$A$ is a set of mutually orthogonal vectors if for all $i \neq j$, $\vec
+		v_{i} \cdot \vec v_{j} = 0$.
+		\begin{enumerate}
+			\item Suppose $\mathcal{A}$ is a set of mutually orthogonal vectors.
+				Is $\mathcal{A}$ a linearly independent set? Why or Why not?
+
+			\item Suppose $\mathcal{A}$ is a set of mutually orthogonal and non-zero
+				vectors. Is $\mathcal{A}$ a linearly independent set? Why or Why
+				not?
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:226]: You're missing the `\end{enumerate}`

File: problemsets-mat223/modules/module4-exercises.tex [hunk 4]
```diff
@@ -181,5 +181,49 @@
 				real numbers.
 			\end{enumerate}
 		\end{solution}
+	
+		\prob Let $\vec u, \vec v \in \R^{n}$. In this problem, we will prove
+		$\Abs{\vec u \cdot \vec v}\leq \Norm{\vec u}\Norm{\vec v}$, the \emph{Cauchy-Schwarz
+		inequality}.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:187]: -> this is called the Cauchy-Schwarz inequality

File: problemsets-mat223/modules/module4-exercises.tex [hunk 5]
```diff
@@ -181,5 +181,49 @@
...
+		\begin{enumerate}
+			\item Warm up: Suppose $\vec u, \vec v \in \R^{2}$ and $\vec u \cdot
+				\vec v = \Norm{\vec u}\Norm{\vec v}\cos \theta$ where $\theta$ is
+				the angle between $\vec u$ and $\vec v$. Show that
+				$\Abs{\vec u \cdot \vec v}\leq \Norm{\vec u}\Norm{\vec v}$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:192]: Rephrase this as: Assuming the geometric definition $\vec u\cdot vec v=...$, prove the Cacuhy-Schwarz inequality.

File: problemsets-mat223/modules/module4-exercises.tex [hunk 6]
```diff
@@ -181,5 +181,49 @@
...
+
+			\item The result is immediate if one (or both) of $\vec u, \vec v$
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:194]: Make this: The Cachy-Schwartz inequality can also be proved using \emph{only} the algebraic definition of the dot product. Keep in mind the following facts which come from the algebraic definition: ... <list the linearity and norm properites you need>

Move the rest to subparts. Make the `u` or `v` = `0` case the first part.

File: problemsets-mat223/modules/module4-exercises.tex [hunk 7]
```diff
@@ -181,5 +181,49 @@
...
+				$\Abs{\vec u \cdot \vec v}\leq \Norm{\vec u}\Norm{\vec v}$, the \emph{Cauchy-Schwarz
+				inequality}. Hint: Consider $P(\frac{b}{2a})$.
+
+			\item Prove $\Abs{\vec u \cdot \vec v}= \Norm{\vec u}\Norm{\vec v}$
+				if the vectors $\vec u, \vec v$ are scalar multiples of each
+				other.
+		\end{enumerate}
+
+		\prob Let $\vec u, \vec v \in \R^{n}$. Prove $\Norm{\vec u + \vec v}\leq
+		\Norm{\vec u}+ \Norm{\vec v}$, the \emph{triangle inequality} in
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:214]: -> This is called the *Triangle Inequality*.

Let's add three parts. First one: draw a picture and explain why its called the triangle inequality. Second one: express `||u+v||` in terms of dot products and square roots. Third part: prove the Triangle inequality.

File: problemsets-mat223/modules/module4-exercises.tex [hunk 8]
```diff
@@ -181,5 +181,49 @@
...
+		$\R^{n}$. Hint: use the \emph{Cauchy-Schwarz inequality}.
+
+		\prob Let $\mathcal{A}= \{\vec v_{1},\dots , \vec v_{k}\} \subset \R^{n}$.
+		$A$ is a set of mutually orthogonal vectors if for all $i \neq j$, $\vec
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:218]: Add a `we have` after the comma. It can be hard to parse a sentence when there is a comma separating the math with no words inbetween.

File: problemsets-mat223/modules/module4-exercises.tex [hunk 9]
```diff
@@ -181,5 +181,67 @@
...
+		\begin{enumerate}
+			\item Assuming the geometric definition
+				$\vec u \cdot \vec v = \Norm{\vec u}\Norm{\vec v}\cos \theta$
+				where $\theta$ is the angle between $\vec u$ and $\vec v$, prove
+				the \emph{Cauchy-Schwarz inequality}.
+
+			\item The \emph{Cauchy-Schwartz inequality} can also be proved using
+				\emph{only} the algebraic definition of the dot product. Keep in
+				mind the following facts which come from the algebraic definition:
+
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:197]: Where did the list of facts go?

File: problemsets-mat223/modules/module4-exercises.tex [hunk 10]
```diff
@@ -181,5 +181,67 @@
...
+				\begin{enumerate}
+					\item Explain why the result is immediate if one (or both)
+						of $\vec u, \vec v$ is the zero vector.
+
+					\item Assume $\vec u, \vec v$ are non-zero vectors. Consider
+						the function $P: \R \to \R$ where $P(t) = \Norm
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:203]: Let's call it $f$ to make the students more comfortable.

File: problemsets-mat223/modules/module4-exercises.tex [hunk 11]
```diff
@@ -181,5 +181,67 @@
...
+						if the vectors $\vec u, \vec v$ are scalar multiples of
+						each other.
+				\end{enumerate}
+		\end{enumerate}
+
+		\prob Let $\vec u, \vec v \in \R^{n}$. In this problem, we will prove
+		$\Norm{\vec u + \vec v}\leq \Norm{\vec u}+ \Norm{\vec v}$. This is called the
+		\emph{Triangle Inequality}.
+		\begin{enumerate}
+			\item Draw a picture and explain why it is called the \emph{Triangle Inequality}.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module4-exercises.tex:227]: -> Pick your favorite vectors `u` and `v` and draw a picture of `u`, `v`, and `u+v`. Root your vectors so that they form the edges of a triangle. Can you explain...

File: problemsets-mat223/modules/module4-exercises.tex (hunk 9 above)
Comment (qrn99) [reply problemsets-mat223/modules/module4-exercises.tex:197]: facts such as:  `$\Norm{\vec a}^2 = \vec a \cdot \vec a`?

Comment (siefkenj) [reply problemsets-mat223/modules/module4-exercises.tex:197]: Yes, and distributivity. You have a colon which introduces nothing....I expected the facts to follow.
